packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -98,4 +117,48 @@ contract SuperchainWETH is WETH98, IERC7802, ISemver {
         return _interfaceId == type(IERC7802).interfaceId || _interfaceId == type(IERC20).interfaceId
             || _interfaceId == type(IERC165).interfaceId;
     }
+
+    /// @notice Sends ETH to some target address on another chain.
+    /// @param _to       Address to send tokens to.
+    /// @param _chainId  Chain ID of the destination chain.
+    function sendETH(address _to, uint256 _chainId) external payable returns (bytes32 msgHash_) {
+        if (_to == address(0)) revert ZeroAddress();
+
+        if (IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).isCustomGasToken()) {
+            revert NotCustomGasToken();
+        }
+
+        IETHLiquidity(Predeploys.ETH_LIQUIDITY).burn{ value: msg.value }();
```

Comment:
beware that this interface might change to `deposit/withdraw`. could we leave a comment so we don;t forget?

Reply:
is a comment necessary? once the interface is updated, this will need to be updated in order for the contract to build, right?

Reply:
since this function-name change is planned, and external devs might read this and start forming a mental model of the architecture, I rather have it documented on the code.
a simple: `// NOTE:  'burn' will soon change to 'deposit'` and the same on mint

Reply:
done.